textrepo-app/src/main/java/nl/knaw/huc/resources/rest/DocumentFilesResource.java

```diff
@@ -45,15 +51,23 @@ public DocumentFilesResource(
   @GET
   @Produces(APPLICATION_JSON)
   @ApiOperation(value = "Retrieve document files")
-  @ApiResponses(value = {@ApiResponse(code = 200, response = ResultPage.class, message = "OK")})
+  @ApiResponses(value = {@ApiResponse(code = 200, response = ResultTextRepoFilePage.class, message = "OK")})
   public Response getDocumentFiles(
-      @PathParam("docId") @Valid UUID docId,
-      @BeanParam FormPageParams pageParams
+      @PathParam("docId")
+      @ApiParam(required = true, example = "34739357-eb75-449b-b2df-d3f6289470d6")
+      @Valid
+          UUID docId,
+      @QueryParam("typeId")
+      @ApiParam(example = "1")
```

Comment:
Expliciet `required = false`, ook al is dat de default, om hier duidelijk aan te geven dat het de bedoeling is dat `typeId` leeg/null mag zijn?

Reply:
De default wordt meegenomen in swagger documentatie, waardoor voor gebruiker duidelijk is dat false de default is

Reply:
+1